enterprise/api/v1/views.py

```diff
@@ -229,13 +232,17 @@ def enterprise_learners(self, request, pk):
             }
         )
         if serializer.is_valid(raise_exception=True):
-            singular_email = serializer.validated_data.get('email')
-            emails = set()
             already_linked_emails = []
             duplicate_emails = []
             errors = []
+            enrolled_count = 0
+
+            singular_email = serializer.validated_data.get('email')
```

Comment:
Since this no longer represents a singular email, can you please just remove this temp variable?